Payjoin-spec.md

```diff
@@ -70,6 +70,42 @@ There is no guarantee concerning the ordering of outputs and inputs in either th
 Note: We strongly recommend the sender to submit a PSBT in Base64 format.
 However, we also accept the PSBT or a bitcoin transaction in hex format.
 
+### Optional parameters
+
+When the payjoin sender sends the POST request to the payjoin endpoint, he can optionally specifies the following parameters:
```

Comment:
```suggestion
When the payjoin sender sends the POST request to the payjoin endpoint, he can optionally specify the following parameters:
```

Reply:
TIL of the github suggestion comment. Amazing